organizations/organizations.go

```diff
@@ -246,3 +246,40 @@ func UpdateLogo(w http.ResponseWriter, r *http.Request) {
 
 	utils.GetSuccess("organization logo updated successfully", nil, w)
 }
+
+func UpdateSettings(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	orgId := mux.Vars(r)["id"]
+	requestData := make(map[string]interface{})
```

Comment:
Would this not affect the structure of the settings schema? Take for example, this update will pass when any json body is passed even if it doesn't conform to what the settings schema should look like. One can pass {"school":"Unilag"} and the entire value of the settings would change pass especially using an updateorcreate query. I think we need to determine what our settings schema is suppose to look like and build a struct for it.